<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mercado</class>
 <widget class="QDialog" name="Mercado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_ABB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Arbol ABB</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caracteristicas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_caracteristicas_ABB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen_ABB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comprar_ABB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_AVL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>191</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Arbol AVL</string>
   </property>
   <widget class="QLabel" name="label_caracteristicas_AVL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caracteristicas:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comprar_AVL">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen_AVL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_ARJ">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>191</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Arbol Roji-Negro</string>
   </property>
   <widget class="QLabel" name="label_caracteristicas_ARJ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caracteristicas:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comprar_ARJ">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen_ARJ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Heap">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>191</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Arbol Heap</string>
   </property>
   <widget class="QLabel" name="label_caracteristicas_Heap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caracteristicas:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comprar_Heap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen_Heap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_espantapajaros">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>181</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Espantapajaros</string>
   </property>
   <widget class="QPushButton" name="pushButton_comprar_espantapajaros">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen_espantapajaros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Costo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_costo_espantapajaros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plata:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_plata">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
